pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -196,6 +212,12 @@ fn create_bm25(
         _ => bail!("could not select key field type and type oid"),
     };
 
+    let predicate_where_escape = if !predicate_where.is_empty() {
+        predicate_where.replace('\'', "''")
```

Comment:
I'm encountering an escape problem here. Perhaps there's a better way to handle it.

Reply:
What's the escape problem?

Reply:
It'a escape character problem. predicate like  `where b = 'str' ` , it will be part of format_hybrid_function argument `function_body`.And then handle by pgsql function. Character ' need to be -> '' (escape). So here I simply replace '  to ''.(sorry for my poor English, hope you could understand )

Reply:
Aren't you already escaping the string passed to `predicates`?
```sql
   predicates => 'category = ''Electronics'''
```

Wouldn't this replacement result in 
```rust
   "category = ''''Electronics''''"
```

Reply:
> Aren't you already escaping the string passed to `predicates`?
> 
> ```sql
>    predicates => 'category = ''Electronics'''
> ```
> 
> Wouldn't this replacement result in
> 
> ```rust
>    "category = ''''Electronics''''"
> ```

I think it when it goes to create_bm25, it already is coverted to 'category = 'Electronics'`   by postgres.